webapi-pip/index.html

```diff
@@ -0,0 +1,119 @@
+    bitmovin.player.core.Player.addModule(window.bitmovin.player['container-mp4'].default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player['container-ts'].default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.mserenderer.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.abr.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.drm.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.xml.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.dash.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.hls.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.crypto.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.style.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.tizen.default);
+    bitmovin.player.core.Player.addModule(window.bitmovin.player.ui.default);
+
+    var APP_ID = 'com.bitmovin'
+    var config = {
+        "key": "---YOUR-KEY-HERE---",
+        playback: {
+            autoplay: true,
+            muted: true
+        },
+        events: {
+            error: function (errObj) {
+                switch (errObj.code) {
+                    default:
+                        console.error('error', errObj);
+                        break;
+                }
+            }
+        },
+    }
+
+    var source = {
+        dash: 'https://bitmovin-a.akamaihd.net/content/MI201109210084_1/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd',
+    };
+
+    var container = document.getElementById('my-player');
+    var player = new bitmovin.player.core.Player(container, config);
+    var videoElement = document.getElementById("videoElement")
+
+    player.setVideoElement(videoElement);
+
+    player.load(source).then(
+        function () {
+            console.log('Successfully created Bitmovin Player instance');
+        },
+        function (reason) {
+            console.log('Error while creating Bitmovin Player instance');
+        }
+    );
+
+    var pipButtonElement = document.getElementById("pipButtonElement");
+    pipButtonElement.addEventListener('click', async function () {
+        try {
+            if (videoElement !== document.pictureInPictureElement) {
+                await videoElement.requestPictureInPicture();
+            } else {
+                await document.exitPictureInPicture();
+            }
+        }
+        catch (error) {
+            logElement = document.getElementById("log");
+            if (!('pictureInPictureEnabled' in document)) {
+                logElement.innerHTML = 'The Picture-in-Picture Web API is not available; Try using Chrome / Safari.';
+            }
+            else if (!document.pictureInPictureEnabled) {
+                logElement.innerHTML = 'The Picture-in-Picture Web API is disabled. Try using Chrome / Safari.';
+            }

```

Comment:
Personally, I'd handle these cases earlier, e.g. either checking the conditions before calling the above functions or maybe even before rendering the button for the user. But I guess this option is fine as well.